dockerfiles/evmapp/entrypoint.sh

```diff
@@ -58,6 +58,17 @@ if [ -z "${SCNODE_NET_DECLAREDADDRESS}" ]; then
 fi
 export SCNODE_NET_DECLAREDADDRESS
 
+#Custom log4j file (optional)
+LOG4J_CUSTOM_CONFIG=""
```

Comment:
I would move this into the beginning of the file where the other empty VARS are defined